wp-content/plugins/mrb-jobs-feed/core.php

```diff
@@ -96,16 +95,119 @@ function single_job_rewrite() {
      * https://codex.wordpress.org/Plugin_API/Action_Reference/admin_post_(action)
      */
     function handle_application_upload() {
-
-        echo "<pre>";
-        var_dump("TODO: handle the job upload");
-        var_dump($_POST); 
-        var_dump($_FILES);
-        var_dump(WP_CONTENT_DIR);
-        var_dump(content_url());
-        echo "</pre>";
-
-
+			if(empty($_POST["verify"])){
+				if(!empty($_POST["your-email"])&&!empty($_POST["your-name"])){
+					$args_newApplicant = array(
+										'post_type' => 'mrb_job_application',
+										'post_ststus' => 'publish',
+										'meta_query' => array(
+											'relation' => 'AND',
+											
+											array(
+											'key' => 'job_id',
+											'value' => $_POST["job_id"] ,
+											'compare'  => '='
+											), array(
+											'key' => 'Email',
+											'value' => $_POST["your-email"] ,
+											'compare'  => '='
+											)
+										)
+										);
+										$new_applicant = new \WP_Query($args_newApplicant);
+										global $wpdb;
+										if($new_applicant->have_posts()){
+											 ?>
+										   <div class="alert alert-danger">
+														<p> youve already applied for this role</p>
+											</div>
+											<?php
+										}
+										else
+										{
+												// Insert post into job application post meta data
+											  $post_id = wp_insert_post(array (
+											   'post_type' => 'mrb_job_application',
+											   'post_title' => "Job application made for ".$_POST["job_title"] ." by ". $_POST["your-name"],
+											   'post_content' => $_POST["your-name"],
+											   'post_status' => 'publish',
+											   'comment_status' => 'closed',   // if you prefer
+											));
+										} 
+				 
+					
+						?>
+						<div class="alert alert-success">
+							<p>New post addded</p>
+						</div>
+					<?php
+				
+				}
+				else{
```

Comment:
I'm struggling to follow these nested conditionals `if/else` statements. Is there a way you can think of to refactor this code?